packages/nextjs/components/scaffold-eth/RainbowKitCustomConnectButton/AddressInfoDropdown.tsx

```diff
@@ -47,7 +47,9 @@ export const AddressInfoDropdown = ({
       <details ref={dropdownRef} className="dropdown dropdown-end leading-3">
         <summary tabIndex={0} className="btn btn-secondary btn-sm pl-0 pr-2 shadow-md dropdown-toggle gap-0 !h-auto">
           <BlockieAvatar address={address} size={30} ensImage={ensAvatar} />
-          <span className="ml-2 mr-1">{displayName}</span>
+          <span className="ml-2 mr-1">
+            {isENS(displayName) ? displayName : address?.slice(0, 5) + "..." + address?.slice(-4)}
```

Comment:
I'm having this runtime error on homepage after a fresh clone and `yarn start`. Adding the import fixed it for me: 
`import { BlockieAvatar, isENS } from "~~/components/scaffold-eth";`

<details>
<summary>Screenshot from error</summary>

![image](https://github.com/scaffold-eth/scaffold-eth-2/assets/55535804/4fc172b0-19ad-4f72-bfa4-c2662efef7bd)
</details>

Also in this line I think there is a typo to match the 4...4 address format, I think it should be `address?.slice(0, 6)`:

```suggestion
            {isENS(displayName) ? displayName : address?.slice(0, 6) + "..." + address?.slice(-4)}
```

Reply:
Oh shoot. I'm sorry. I forgot to test. What a noob!